awxkit/awxkit/api/pages/workflow_job_templates.py

```diff
@@ -42,6 +42,9 @@ def payload(self, **kwargs):
             "ask_inventory_on_launch",
             "ask_scm_branch_on_launch",
             "ask_limit_on_launch",
+            "ask_labels_on_launch",
+            "ask_skip_tags_on_launch",
+            "ask_tags_on_launch",
             "limit",
             "scm_branch",
             "survey_enabled",
```

Comment:
Action item:

Also in this module, the WFJT model has a `add_label` method. The WFJT node should _also_ have this method, because this PR adds labels on it.

Same goes for labels on schedules, the awxkit Schedule model should get a new `add_label` method.

----

Instance groups was also an added endpoint, and `add_instance_group` is a method on some models. This method should be added to WFJT nodes and schedules as well.

Reply:
I'll do that

Reply:
add_label: https://github.com/ansible/awx/pull/12886

Reply:
Resolving from merge